pkg/download/downloader_test.go

```diff
@@ -140,3 +147,132 @@ func collectFiles(t *testing.T, scanPath string) []string {
 	}
 	return outFiles
 }
+
+func Test_extractContentType(t *testing.T) {
+	type args struct {
+		file string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "type zip",
+			args: args{
+				file: filepath.Join(testdataPath(), "test-with-directory.zip"),
+			},
+			want:    "application/zip",
+			wantErr: false,
+		},
+		{
+			name: "type tar.gz",
+			args: args{
+				file: filepath.Join(testdataPath(), "test-with-nesting-with-directory-entries.tar.gz"),
+			},
+			want:    "application/x-gzip",
+			wantErr: false,
+		},
+		{
+			name: "type elf-bin",
```

Comment:
this test seems unnecessary to me.

Reply:
We need this later in #45 . This verifies the content type of direct downloads. I'm not sure if this testing should be done by us.
We assume that a binary file gets the type `application/octet-stream`. This test makes sure that our assumptions are correct.